resources/js/Components/MismatchesTable.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <wikit-table>
+        <thead>
+            <tr>
+                <th>{{$i18n('column-property')}}</th>
+                <th>{{$i18n('column-wikidata-value')}}</th>
+                <th>{{$i18n('column-external-value')}}</th>
+                <th>{{$i18n('column-upload-info')}}</th>
+            </tr>
+        </thead>
+        <tbody>
+            <mismatch-row v-for="mismatch in mismatches" :key="mismatch.id" :mismatch="mismatch" />
```

Comment:
It will not. When looping through items in a list to generate DOM nodes, reactive frameworks such as vue need to know what to do with the Virtual DOM tree in case the list changes. You can read more at: https://vuejs.org/v2/guide/list.html#Maintaining-State

Reply:
thank you!